src/main/kotlin/model/Player.kt

```diff
@@ -0,0 +1,14 @@
+package model
+
+import view.InputView
+
+class Player(name: String) : BasePlayer(name) {
+    override fun makeDecision(): Boolean {
+        val decision = InputView.requestPlayerDecision(name)
```

Comment:
Is it good practice to place view logic in the model?
I'd recommend keeping the model independent, since it contains business or domain logic.